source/administration/changelog.md

```diff
@@ -4,6 +4,81 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+ - Fixed an issue where bulk import rejected team names prefixed with reserved keywords, even with additional text appended.
+ - Fixed an issue where System Admin could no longer manage custom emoji after running ``bin/mattermost permissions reset``.
+ - Fixed an issue where a user's role in Team Members dialog did not update when a user was searching for the user.
+ - Fixed an issue where Bleve was not correctly setting the query size, missing search results.
+ - Fixed an issue where the timezone count was not displayed correctly when a user set a new timezone and then changed it to set automatically.
+ - Fixed an issue where existing users were not shown in the Invite Members flow.
+ - Fixed an issue where the **System Console > User Management > Users** page was too tall and the **Revoke All Sessions** button was cut off when a license banner was present.
+ - Fixed an issue where the **Email verified** banner was red instead of green.
+ - Fixed an issue where **Copy Theme Colors** button in **Account Settings > Display > Theme** was not themed correctly.
+ - Fixed an issue where archived channel icons were too dark in the Channel Info modal with the Dark Theme.
+ - Fixed an issue where the **Save** button was not visible in browser for Safari on iPad device.
+ - Fixed an issue where the thumbnail of a user was not displayed correctly when searching for a Direct Message channel.
+ - Fixed an issue where text flowed outside the "Invite Members" button in "Invite People" page for some languages.
+ - Fixed an issue where the **Next** button in **Main Menu > Manage Members** was not visible to be able to see the last few members of the team.
+ - Fixed an issue where a different behavior was seen when pasting a table into message compose and to message edit box.
+ - Fixed an issue where one-byte unicode emoji did not support skin tones.
+ - Fixed an issue where no error was reported in server logs if a plugin icon was invalid.
+ - Fixed an issue where providing AutocompleteData did not log a proper error in the System Console.
+ - Fixed an issue where signup password minimum length error messages were inconsistent.
+ - Fixed an issue where the right-hand side overlapped the GitHub Plugin tooltip.
+ - Fixed an issue where the plugin right-hand side did not show tooltips when a user hovered over the Close or Expand/Shrink icons.
+ - Fixed an issue where query string parameters were omitted from interactive dialog request urls.
+ - Fixed an issue where ``store.GetPostsSince()`` did not sanitise deleted posts.
+ - Fixed an issue with a panic caused by nil pointer dereference in ``importTeam``.
+ 
+### config.json
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
```

Comment:
Will update once I get details from Martin on config.json, Database Changes, API Changes and Websocket Event Changes.